src/vaadin-pro-grid-edit-column-text-field.html

```diff
@@ -26,6 +26,11 @@
       static get is() {
         return 'vaadin-pro-grid-edit-column-text-field';
       }
+
+      ready() {
+        super.ready();
+        this.setAttribute('theme', 'pro-grid-editor');
```

Comment:
One important question: should we consider applying this theme to the generic `vaadin-text-field` instead of its extension?

This would enable us to provide components like `vaadin-combo-box` and `vaadin-date-picker`, and ensure they have roughly the same appearance as the text-field does, using theme attribute propagation feature. 

On the other hand, the components like `vaadin-number-field` would be unaffected, so there would be still an inconsistence regarding the appearance of Vaadin components when used as cell editors.

Reply:
Hmm...that would make it a public variant. Maybe we could consider this later?

Reply:
Yes, let's reconsider this later if we start receiving this kind of feedback.